### PR TITLE
use dependabot to update containers

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -3,10 +3,17 @@ updates:
...
-    time: "03:00"
-    timezone: Europe/Paris
+    interval: daily
+    time: "12:00"
+  open-pull-requests-limit: 10
+  labels:
+  - 3. to review
+  - dependencies
+- package-ecosystem: "docker"
+  directory: "/Containers/"
```
Comment (szaimen) [inline .github/dependabot.yml:13]: @juliushaertl do you know if that will pick up all dockerfiles recursively?

**Review by juliushaertl — APPROVED** [review]
Let's give it a try I'd say ;)

**Comment by szaimen** [comment]
Thanks for the feedback, Julius! Lets merge this after I published v0.3.0

File: .github/dependabot.yml (hunk 1 above)
Comment (szaimen) [reply .github/dependabot.yml:13]: okay thanks

**Pull request merged**